src/graphics/device.js

```diff
@@ -1839,6 +1858,46 @@ pc.extend(pc, function () {
 
         /**
          * @function
+         * @name pc.GraphicsDevice#setTransformFeedbackBuffer
+         * @description Sets the output vertex buffer. It will be written to by a shader with transform feedback varyings.
+         * @param {pc.VertexBuffer} tf The output vertex buffer
+         */
+        setTransformFeedbackBuffer: function (tf) {
+            if (this.transformFeedbackBuffer !== tf) {
```

Comment:
Just to simplify this method for reading and avoiding extra indention, consider such style:
```javascript
setTransformFeedbackBuffer: function (tf) {
    if (this.transformFeedbackBuffer === tf)
        return;

    this.transformFeedbackBuffer = tf;
    
    if (this.webgl2) {
        var gl = this.gl;
        if (tf) {
            gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, this.feedback);
        } else {
            gl.bindTransformFeedback(gl.TRANSFORM_FEEDBACK, null);
        }
    }
},
```